src/UI/Areas/Identity/Pages/Account/Manage/EnableAuthenticator.cshtml.cs

```diff
@@ -102,7 +102,8 @@ internal class EnableAuthenticatorModel<TUser> : EnableAuthenticatorModel where
             }
 
             await _userManager.SetTwoFactorEnabledAsync(user, true);
-            _logger.LogInformation("User with ID '{UserId}' has enabled 2FA with an authenticator app.", user.Id);
+            var userId = await _userManager.GetUserIdAsync(user);
```

Comment:
Doing these extra calls for logging seems overkill. We can do the `LoggerMessage.Define` pattern for all the logging here.

Reply:
Your call. But file a bug and do it on a separate PR.

Reply:
I will file a separate issue for this.